frontend/src/views/WheresMyGene/common/store/reducer.ts

```diff
@@ -38,6 +39,7 @@ export interface State {
   compare?: CompareId;
   xAxisHeight: number;
   filteredCellTypes: string[];
+  filteredCellTypeIds: string[];
```

Comment:
i considered updating `filteredCellTypes` from being a list of the cell type names to being a list of `CellType` entities, but i think it would have been annoying to update every existing use case (and future use cases) to filter it out into either a list of names or a list of ids